### PR TITLE
store source nodes as columns and destination nodes as rows

### PR DESCRIPTION
**Pull request by swilly22**

### CODE REVIEW

File: src/execution_plan/ops/op_conditional_traverse.c [hunk 1]
```diff
@@ -57,16 +55,15 @@ OpResult CondTraverseConsume(OpBase *opBase, QueryGraph* graph) {
     }
 
     /* Get node from current column. */
```
Comment (jeffreylovitz) [inline src/execution_plan/ops/op_conditional_traverse.c]: This comment should say "row".

File: src/arithmetic/algebraic_expression.c [hunk 2]
```diff
@@ -137,6 +137,21 @@ static inline void _AlgebraicExpression_Execute_MUL(GrB_Matrix C, GrB_Matrix A,
...
 }
 
+// Reverse order of operand within expression,
+// A*B*C will become C*B*A. 
+void _AlgebraicExpression_ReverseOperandOrder(AlgebraicExpression *exp) {
+    int right = exp->operand_count-1;
+    int left = 0;
+    while(right > left) {
+        AlgebraicExpressionOperand leftOp = exp->operands[left];
+        AlgebraicExpressionOperand rightOp = exp->operands[right];
```
Comment (jeffreylovitz) [inline src/arithmetic/algebraic_expression.c]: Only 1 of these strictly needs to be saved in a temporary variable.

File: src/execution_plan/ops/op_conditional_traverse.c (hunk 1 above)
Comment (swilly22) [reply src/execution_plan/ops/op_conditional_traverse.c]: We're picking nodes along the current column. whenever the column has NNZ in it.

Comment (jeffreylovitz) [reply src/execution_plan/ops/op_conditional_traverse.c]: Ah, I see - you're right, the original makes sense!

**Pull request merged**